.gitignore

```diff
@@ -1,4 +1,5 @@
 .gradle
+**/**/build/
 /build/
```

Comment:
Can consolidate both of these entries by replacing them with **/build/.  Might also want to add **/bin/ as well.

Reply:
removed /build/ and added **/bin/**